resources/providers/awslib/ecr_provider.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/ecr/types"
 )
 
+type EcrRepository types.Repository
```

Comment:
nit: isn't the `EcrRepositories` an array of `EcrRepository`?

Do we need them both?